### ISSUE TITLE
Support for Supabase

### ISSUE THREAD

**Issue by 1saifj** [vyuh-tech/vyuh#12]
Hey,
Thanks for this great project, can we support Supabase integration?

**Comment by pavanpodila** [comment]
Yup that's on the roadmap. Which service are you keen on using in Supabase

**Comment by 1saifj** [comment]
Thx, looking forward to see it in your project.
the database and auth could be enough for me.
